crates/bevy_core_pipeline/src/core_3d/camera_3d.rs

```diff
@@ -20,11 +20,33 @@ use serde::{Deserialize, Serialize};
 #[reflect(Component)]
 pub struct Camera3d {
     /// The clear color operation to perform for the main 3d pass.
+    ///
+    /// **Note:** When [`Camera3d::screen_space_specular_transmission_steps`] > `0`, a fully opaque clear
+    /// color will “cover” the environment map light's texture, preventing it from
+    /// being refracted “through” transmissive objects, even if there are no opaque
+    /// objects behind them. To work around this issue, consider using a clear color
+    /// with an alpha value of `0.0`.
```

Comment:
Hmm... I don't think we should change the default clear color, as depending on the platform and window settings (and I think by default for `canvas` on the web), this could result in a semitransparent window which could be surprising/undesirable.

The main reason is only to ensure the environment map can cover portions of the transmitted image that would otherwise be obscured by the clear color. This only really makes sense in test scenes without a skybox or a significant scene that would cover the entire viewport.

Reply:
if 0 alpha has bad side effects maybe we shouldn't suggest it. maybe better to suggest a skybox instead?

Reply:
A bit confusing, but I don't feel like we need to block on improving this comment.

Reply:
I moved this to `screen_space_specular_transmission_steps`, and entirely rephrased it, so I believe it now makes more sense in context. I also added a note about the potential issues of using transparent clear colors:

```rust
    /// How many individual steps should be performed in the [`Transmissive3d`](crate::core_3d::Transmissive3d) pass.
    ///
    /// Roughly corresponds to how many “layers of transparency” are rendered for screen space
    /// specular transmissive objects. Each step requires making one additional
    /// texture copy, so it's recommended to keep this number to a resonably low value. Defaults to `1`.
    ///
    /// ### Notes:
    /// - No copies will be performed if there are no transmissive materials currently being rendered,
    ///   regardless of this setting.
    /// - Setting this to `0` disables the screen-space refraction effect entirely, and falls
    ///   back to refracting only the environment map light's texture.
    /// - If set to more than `0`, any opaque [`clear_color`](Camera3d::clear_color) will obscure the environment
    ///   map light's texture, preventing it from being visible “through” transmissive materials. If you'd like
    ///   to still have the environment map show up in your refractions, you can set the clear color's alpha to `0.0`.
    ///   Keep in mind that depending on the platform and your window settings, this may cause the window to become
    ///   transparent.
```